src/Simplic.MessageBroker.RabbitMQ/UnityContainerServerExtensions.cs

```diff
@@ -57,12 +60,22 @@ ISessionService sessionService
                             var queueName = ((QueueAttribute)attributes[0]).Name;
                             consumers.Add(queueName, consumerType);
                         }
+
+                        attributes = consumerType.GetCustomAttributes(typeof(NoQueueAttribute), true);
+                        if (attributes.Any())
+                            queuelessConsumer.Add(consumerType);
                     }
 
                     Console.WriteLine($"Consumers found: {consumers.Count}");
 
+                    Console.WriteLine($"Queueless consumers found: {queuelessConsumer.Count}");
```

Comment:
```suggestion
                    Console.WriteLine($"Consumers found: {consumers.Count}");
                    Console.WriteLine($"Queueless consumers found: {queuelessConsumer.Count}");
```
I think the newline is a bit confusing in that case since both lines do the same thing